pkg/cache/external/scheduler_cache.go

```diff
@@ -603,46 +603,24 @@ func (cache *SchedulerCache) IsPVCUsedByPods(key string) bool {
 	return ok
 }
 
-func (cache *SchedulerCache) maxNodeGeneration() int64 {
-	var maxGeneration int64
-	for _, node := range cache.nodesMap {
-		if maxGeneration < node.Generation {
-			maxGeneration = node.Generation
+func (cache *SchedulerCache) updatePVCRefCounts(node *framework.NodeInfo, removeNode bool) {
+	for k, v := range cache.pvcRefCounts {
+		delete(v, node.Node().Name)
```

Comment:
Save node.Node().Name to a variable at the top of the function so it can be re-used.

Reply:
Updated it. Thank you!